charts/fluentd/values.yaml

```diff
@@ -290,100 +290,108 @@ dashboards:
 plugins: []
 # - fluent-plugin-out-http
 
+defaultConfigs:
+  prometheusConf:
+    enabled: true
```

Comment:
Same thing with what i said. No sense to have to fix this in two places. Maybe disable by default then.

This means, dropping this `defaultConfigs` dict and setting
```yaml
configMapConfigs: []
```